app/components/about-component.tsx

```diff
@@ -0,0 +1,25 @@
+interface AboutProps {
+  image?: string;
```

Comment:
since we allow the props to be undefined, we should set some defaults for all the props within the component